cmd/bluesky/main.go

```
@@ -61,28 +61,53 @@ func Handler(ctx context.Context) error {
+	err = processRSSFeed(scrapeCtx, cfg, db, bsky, cfg.NewsBlogRSSFeed.URL, "-newsblog")
+	if err != nil {
+		log.Error().Err(err).Msg("error scraping AWS news blog feed")
+		return err
+	}
+	return nil
+}
+
 func main() {
-	lambda.Start(Handler)
+	//lambda.Start(Handler)
+	Handler(context.Background())
```

golangci-lint

Error return value is not checked

(errcheck)